src/index.tsx

```diff
@@ -128,6 +128,11 @@ export class LikeCoinWalletConnector {
           : true,
 
       language: options.language || 'en',
+
+      // Events
+      onToggleCollapsibleConnectionMethodList:
+        options.onToggleCollapsibleConnectionMethodList || (() => {}),
+      onSelectConnectionMethod: options.onSelectConnectionMethod || (() => {}),
```

Comment:
Maybe we should have one function that receive all the analytics event callback, instead of defining them one by one?

Reply:
Updated